lua/lazyvim/plugins/extras/util/rest.lua

```diff
@@ -0,0 +1,24 @@
+vim.filetype.add({
+  extension = {
+    ["http"] = "http",
+  },
+})
+return {
+  {
+    "mistweaverco/kulala.nvim",
+    ft = "http",
+    keys = {
+      { "<leader>rs", "<cmd>lua require('kulala').run()<cr>", desc = "Send the request" },
+      { "<leader>rt", "<cmd>lua require('kulala').toggle_view()<cr>", desc = "Toggle headers/body" },
+      { "<leader>rp", "<cmd>lua require('kulala').jump_prev()<cr>", desc = "Jump to previous request" },
+      { "<leader>rn", "<cmd>lua require('kulala').jump_next()<cr>", desc = "Jump to next request" },
```

Comment:
There's also the [refactoring Extra](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/editor/refactoring.lua) which uses `<leader>r` keymaps. Do you think it would be possible to change these keymaps to `<leader>R` instead, so there won't be any confusion? It would also be great if you could put something like 
```lua
  { "<leader>R", "", desc = "+Rest" },
```
so it would be picked up and showed with a description in the `which-key` menu?

Reply:
sure.

Reply:
this may not qualify as an extra due to the simplicity but it does improve qol :)

Reply:
It's up to the maintainer to decide. I'm just a simple user like you. Good luck :)

Reply:
Thanks for the help!